policyengine_us/variables/gov/states/co/cdhs/tanf/co_tanf.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class co_tanf(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "CO TANF"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "co_tanf_eligible"
+
+    def formula(spm_unit, period, parameters):
+        payment_standard = spm_unit("co_tanf_payment_standard", period)
```

Comment:
To fix [this issue](https://github.com/PolicyEngine/policyengine-us/actions/runs/3962551774/jobs/6789353964#step:5:473)
```suggestion
        grant_standard = spm_unit("co_tanf_grant_standard", period)
```